mobile/ios/lightning/Supporting/EXBuildConstants.json

```diff
@@ -1 +1 @@
-{"EXPO_RUNTIME_VERSION":"2.8.4","STANDALONE_CONTEXT_TYPE":"user","DEFAULT_API_KEYS":{"GOOGLE_MAPS_IOS_API_KEY":"","AMPLITUDE_KEY":"1e246ef3dacaabe8648768d7c35fceb1"},"developmentUrl":"expcce5b8de9dc3494199f753d071f82699://192.168.14.254:19000"}
\ No newline at end of file
+{"EXPO_RUNTIME_VERSION":"2.8.4","STANDALONE_CONTEXT_TYPE":"user","DEFAULT_API_KEYS":{"GOOGLE_MAPS_IOS_API_KEY":"","AMPLITUDE_KEY":"1e246ef3dacaabe8648768d7c35fceb1"},"developmentUrl":"expcce5b8de9dc3494199f753d071f82699://192.168.30.211:19000"}
```

Comment:
Do iOS devs usually gitignore this file? Seems like it just changed the IP address during local development.

Reply:
I'm not sure, but doesn't look like it: https://medium.com/@jalalazimi/gitignore-for-react-native-project-fc1555c6af6b